plugins/steelseries/steelseries.quirk

```diff
@@ -41,7 +46,7 @@ Plugin = steelseries
 GType = FuSteelseriesFizz
 Name = Aerox 3 Wireless Mouse
 Icon = input-mouse
-CounterpartGuid = USB\VID_1038&PID_183B
+CounterpartGuid = HIDRAW\VEN_0111&DEV_183A
```

Comment:
However, is is possible to manager multiple `CounterpartGuid` in the quirk file?

```
CounterpartGuid = USB\VID_1038&PID_183B,HIDRAW\VEN_0111&DEV_183A
```
or
```
CounterpartGuid = USB\VID_1038&PID_183B HIDRAW\VEN_0111&DEV_183A
```
or even
```
CounterpartGuid = USB\VID_1038&PID_183B
CounterpartGuid = HIDRAW\VEN_0111&DEV_183A
```

Reply:
It's probably a comma, but check fu-device.c as I don't think anyone has tried that before.

Reply:
not working; making a commit to support coma separated list of GUIDs.